Content.Trauma.Common/Botany/SharedPlantAnalyzerComponent.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Content.Trauma.Common.Botany;
+public enum HarvestType : byte
```

Comment:
change the upstream code to use this, remove the enum from Content.Server.Botany and replace it with `// Trauma - moved HarvestType to Content.Trauma.Common.Botany`